analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

```diff
@@ -382,12 +382,32 @@ histologies <-
   dplyr::group_by(short_histology) %>% 
   dplyr::mutate(group_n = dplyr::n()) %>% 
   # Ungroup the data
-  dplyr::ungroup() %>%
+  dplyr::ungroup() %>% 
+  # Temporarily we will put the n = 1 `short_histology` samples in the `Other` group. 
+  #TODO: Remove lines 387 - 400 when this is split into two panels
+  dplyr::mutate(
+    short_histology = dplyr::case_when(
+    group_n == 1 ~ "Other", 
+    TRUE ~ as.character(short_histology))) %>% 
+  dplyr::mutate(
+    sample_color = dplyr::case_when(
+    group_n == 1 ~ "#cc0052",
+    TRUE ~ as.character(sample_color))) %>% 
```

Comment:
Use the same method to reset the colors as before, for flexibility and robustness
```suggestion
  dplyr::mutate(sample_color = dplyr::recode(
    short_histology,
    !!!histology_col_palette
  )) %>%
```